### PR TITLE
Draft: fix ScaleMultiplier on annotation objects

### PR DESCRIPTION
**Pull request by carlopav**
Fixed regression for previously introduced ScaleMultiplier property

Thank you for creating a pull request to contribute to FreeCAD! To ease integration, please confirm the following:

- [ ] Branch rebased on latest master `git pull --rebase upstream master`
- [ ] Unit tests confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [ ] Commit message is [well-written](https://chris.beams.io/posts/git-commit/)
- [ ] Commit message includes `issue #<id>` or `fixes #<id>` where `<id>` is the [associated MantisBT](https://freecadweb.org/wiki/tracker#GitHub_and_MantisBT) issue id if one exists

And please remember to update the Wiki with the features added or changed once this PR is merged.  
**Note**: If you don't have wiki access, then please mention your contribution on the [0.19 Changelog Forum Thread](https://forum.freecadweb.org/viewtopic.php?f=10&t=34586).

---

### CODE REVIEW

**Comment by carlopav** [comment]
@yorikvanhavre now it's ready to merge:
i reverted your fix because Scalemultiplier property, if not present is created onDocumentRestored.

File: src/Mod/Draft/draftobjects/draft_annotation.py [hunk 1]
```diff
@@ -42,6 +42,18 @@ def __init__(self, obj, tp="Annotation"):
         self.Type = tp
 
 
+    def onDocumentRestored(self, obj):
+        if hasattr(obj, "ViewObject"):
```
Comment (yorikvanhavre) [inline src/Mod/Draft/draftobjects/draft_annotation.py:46]: if this is run on console mode (no GUI), the obj will have a "ViewObject" attribute, but obj.ViewObject  is None. So the next line will fail as None has no PropertiesList attribute. so I think the above line should be:
`if hasattr(obj, "ViewObject") and obj.ViewObject:`

Comment (carlopav) [reply src/Mod/Draft/draftobjects/draft_annotation.py:46]: thanks, keep learning :)

Comment (carlopav) [reply src/Mod/Draft/draftobjects/draft_annotation.py:46]: actually I really prefer the way you did in Arch: a separated `set_properties()` method that is called `onDocumentRestored` and checks if all the properties are present...

**Comment by carlopav** [comment]
done!

**Comment by carlopav** [comment]
🤞

**Pull request merged**